main.go

```diff
@@ -1,11 +1,5 @@
 package main
 
-func Divine()  {
-	
-}
 func main() {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`Add` is unused (deadcode)